package.json

```diff
@@ -7,7 +7,7 @@
     "start": "ng serve",
     "prebuild:tsnode": "ts-node git.version.ts",
     "prebuild:downloadcodegen": "wget --no-verbose http://central.maven.org/maven2/org/openapitools/openapi-generator-cli/3.3.1/openapi-generator-cli-3.3.1.jar -O swagger-codegen-cli.jar",
-    "prebuild:generatecode": "java -jar swagger-codegen-cli.jar generate -i https://raw.githubusercontent.com/ga4gh/dockstore/${WEBSERVICE_VERSION}/dockstore-webservice/src/main/resources/swagger.yaml -l typescript-angular -o src/app/shared/swagger -c swagger-config.json", 
```

Comment:
FYI, ran into an issue following the readme. 
This variable doesn't exist outside of Travis-CI.